.github/workflows/NUnitConsoleAndEngine.CI.yml

```diff
@@ -1,118 +1,84 @@
 ﻿name: NUnitConsoleAndEngine.CI
 
 on:
-  push:
-    branches:
-      - main
-      - release
-      - version3x
   pull_request:
-  
+    branches-ignore:
```

Comment:
This is just to match the appveyor config, I can revert this if necessary (but the paths-ignore part can probably stay).